### ISSUE TITLE
Docs wrong?

### ISSUE THREAD

**Issue by thangcoder** [Analog-Labs/testnet_issue_tracker#618]
![Screenshot 2024-06-09 at 15 48 47](https://github.com/Analog-Labs/testnet_issue_tracker/assets/20993232/f73c9d27-99c8-4dbb-812f-dbe041164bc7)

Doc write pull analoglabs/timechain image but run timechain-node so throw error:
Unable to find image 'timechain-node:latest' locally

I think right is:
docker run -d -p 9944:9944 -p 30403:30303 analoglabs/timechain --base-path /data --rpc-external

**Comment by tiaro** [comment]
you're right. But that's not all, many more parameters are still missing:
`docker run -d --mount type=bind,source=/home/analog/data,target=/data -p 9944:9944 -p 30303:30303 --name time-chain analoglabs/timechain \
        --base-path /data \
        --name MYCOOLNODENAME \
        --telemetry-url 'wss://telemetry.analog.one/submit 9' \
        --rpc-external \
        --unsafe-rpc-external`
That's what I suggest
